RXSwiftTest/Extensions/UIViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIViewController.swift
+//  RXSwiftTest
+//
+//  Created by Sunith B on 30/08/19.
+//  Copyright © 2019 Sunith B. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController {
+    
+    func showAlert(title: String, message: String) {
```

Comment:
Duplicate function. You seem to be using the one defined from the Helper class but not this